File: GestionAssuranceFx/src/gestionassurancefx/Entities/Contrat.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package gestionassurancefx.Entities;

import java.sql.Date;

/**
 *
 * @author Ahmed Derbel
 */
public class Contrat {
    int id;
    String nom;
    String description;
    int cin_assure;
    String nomEntr;
    String type;
    int id_type;
    Date date_debut;
    Date date_Echeance;
    int etat;
    float prime;
    public Contrat(){
        
    }

    public Contrat(int id, String nom, String description, int cin_assure,String nomEntr, String type, int id_type,Date date_debut, Date date_Echeance, int etat, float prime) {
        this.id = id;
        this.nom = nom;
        this.description = description;
        this.cin_assure = cin_assure;
        this.nomEntr=nomEntr;
        this.type = type;
        this.id_type=id_type;
        this.date_debut = date_debut;
        this.date_Echeance = date_Echeance;
        this.etat = etat;
        this.prime = prime;
    }

    public int getId_type() {
        return id_type;
    }

    public void setId_type(int id_type) {
        this.id_type = id_type;
    }

    
    
    
    public String getNomEntr() {
        return nomEntr;
    }

    public void setNomEntr(String nomEntr) {
        this.nomEntr = nomEntr;
    }

    
    public int getCin_assure() {
        return cin_assure;
    }

    public void setCin_assure(int cin_assure) {
        this.cin_assure = cin_assure;
    }

    public int getEtat() {
        return etat;
    }

    public void setEtat(int etat) {
        this.etat = etat;
    }

    public float getPrime() {
        return prime;
    }

    public void setPrime(float prime) {
        this.prime = prime;
    }
  
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Date getDate_debut() {
        return date_debut;
    }

    public void setDate_debut(Date date_debut) {
        this.date_debut = date_debut;
    }

    public Date getDate_Echeance() {
        return date_Echeance;
    }

    public void setDate_Echeance(Date date_Echeance) {
        this.date_Echeance = date_Echeance;
    }

    @Override
    public String toString() {
        return "Contrat{" + "id=" + id + ", nom=" + nom + ", description=" + description + ", cin_assure=" + cin_assure + ", nomEntr=" + nomEntr + ", type=" + type + ", date_debut=" + date_debut + ", date_Echeance=" + date_Echeance + ", etat=" + etat + ", prime=" + prime + '}';
    }

   
   
    
    
}

File: Assurance/src/ContratBundle/Entity/Contrat.php
<?php

namespace ContratBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Contrat
 *
 * @ORM\Table(name="contrat")
 * @ORM\Entity
 */
class Contrat
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=30, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="text", length=65535, nullable=true)
     */
    private $description;

    /**
     * @var integer
     *
     * @ORM\Column(name="cin_assure", type="integer", nullable=true)
     */
    private $cinAssure;

    /**
     * @var string
     *
     * @ORM\Column(name="nomEntr", type="string", length=20, nullable=true)
     */
    private $nomentr;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=255, nullable=false)
     */
    private $type;

    /**
     * @var integer
     *
     * @ORM\Column(name="id_type", type="integer", nullable=false)
     */
    private $idType;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_debut", type="datetime", nullable=true)
     */
    private $dateDebut;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_echeance", type="datetime", nullable=true)
     */
    private $dateEcheance;

    /**
     * @var integer
     *
     * @ORM\Column(name="etat", type="integer", nullable=false)
     */
    private $etat;

    /**
     * @var float
     *
     * @ORM\Column(name="prime", type="float", precision=10, scale=0, nullable=false)
     */
    private $prime;

    /**
     * @var integer
     *
     * @ORM\Column(name="nvprime", type="integer", nullable=true)
     */
    private $nvprime;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * @param string $nom
     */
    public function setNom($nom)
    {
        $this->nom = $nom;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return int
     */
    public function getCinAssure()
    {
        return $this->cinAssure;
    }

    /**
     * @param int $cinAssure
     */
    public function setCinAssure($cinAssure)
    {
        $this->cinAssure = $cinAssure;
    }

    /**
     * @return string
     */
    public function getNomentr()
    {
        return $this->nomentr;
    }

    /**
     * @param string $nomentr
     */
    public function setNomentr($nomentr)
    {
        $this->nomentr = $nomentr;
    }

    /**
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType($type)
    {
        $this->type = $type;
    }

    /**
     * @return int
     */
    public function getIdType()
    {
        return $this->idType;
    }

    /**
     * @param int $idType
     */
    public function setIdType($idType)
    {
        $this->idType = $idType;
    }

    /**
     * @return \DateTime
     */
    public function getDateDebut()
    {
        return $this->dateDebut;
    }

    /**
     * @param \DateTime $dateDebut
     */
    public function setDateDebut($dateDebut)
    {
        $this->dateDebut = $dateDebut;
    }

    /**
     * @return \DateTime
     */
    public function getDateEcheance()
    {
        return $this->dateEcheance;
    }

    /**
     * @param \DateTime $dateEcheance
     */
    public function setDateEcheance($dateEcheance)
    {
        $this->dateEcheance = $dateEcheance;
    }

    /**
     * @return int
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * @param int $etat
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;
    }

    /**
     * @return float
     */
    public function getPrime()
    {
        return $this->prime;
    }

    /**
     * @param float $prime
     */
    public function setPrime($prime)
    {
        $this->prime = $prime;
    }

    /**
     * @return int
     */
    public function getNvprime()
    {
        return $this->nvprime;
    }

    /**
     * @param int $nvprime
     */
    public function setNvprime($nvprime)
    {
        $this->nvprime = $nvprime;
    }


}

